.github/workflows/clang-format.yml

```diff
@@ -0,0 +1,40 @@
+name: "Check clang format"
+
+on:
+  push:
+    branches:
+    - 'dev'
+    - 'stable'
+  pull_request:
+    branches:
+    - 'dev'
+    - 'release-*'
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+      with:
+          submodules: recursive
+
+    - name: Update package database
+      run: sudo apt --assume-yes update
+
+    - name: Install wget
+      run: sudo apt --assume-yes install wget
+
+    - name: Install automatic llvm (stable branch)
+      run: sudo bash -c "$(wget -O - https://apt.llvm.org/llvm.sh)"
+
+    - name: Install clang-format-11
+      run: sudo apt --assume-yes install clang-format-11
+
+    - name: Install python
+      run: sudo apt --assume-yes install python3
+
+    - name: Run clang-format
+      run: |
+        alias clang-format=clang-format-11
```

Comment:
I see that there are errors in the clang-format build. Try using `update-alternatives` instead of `alias`.